modular_zubbers/code/datums/bubber_quirks/hydrophilic.dm

```diff
@@ -0,0 +1,12 @@
+/datum/quirk/hydrophilic
+	name = "Hydrophilic"
+	desc = "(Slimeperson only) Your molecules are highly hydrophilic - or, in layman's terms, dissolve very well in water."
+	icon = "bug" // I can't be asked to make an icon for this, my spritework sucks.
+	value = 0
+	quirk_flags = QUIRK_HUMAN_ONLY
```

Comment:
?

Reply:
Oh, I see, this is for the type, not the species. I suppose this prevents it from applying to.... borgs? 🤔

Reply:
Borgs, simplemobs, etc.